samples/file-encryptor/enclave/mbedtls_src/encryptor.cpp

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
-#include "encryptor.h"
+#include <mbedtls/aes.h>
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/entropy.h>
 #include <string.h>
-#include "common.h"
+
+#include "aes_context.h" // Must be included before common/encryptor.h
```

Comment:
Then why not put the include inside common/encryptor.h where some other includes already appear?

Reply:
The intention here is to make the `aes_context` an opaque type in `common/encryptor.h` so that the header can be shared among two implementations.  I think you can just define the struct in this cpp file rather than having an extra header given that `m_aescontext` is used only in this file.

Reply:
removed header aes_context.h and added struct definition in encryption.cpp